### ISSUE TITLE
upgrade node version in install.sh

### ISSUE THREAD

**Issue by prasadtalasila** [prasadtalasila/BITS-Darshini#113]
The default node version installed on Ubuntu is node.js 4.x. Use digital ocean tutorial to update the default installed version to node.js v 6.11.

**Comment by shilparaaju** [comment]
The addition of the commands 

cd ~
curl -sL https://deb.nodesource.com/setup 6.x -o nodesource setup.sh
sudo bash nodesource_setup.sh
sudo apt-get install nodejs -y
sudo apt-get install build-essential

failed to install nodejs.

The following was the error : 
The following packages have unmet dependencies:
 npm : Depends: nodejs but it is not going to be installed
       Depends: node-abbrev (>= 1.0.4) but it is not going to be installed
       Depends: node-ansi (>= 0.3.0-2) but it is not going to be installed
       Depends: node-ansi-color-table but it is not going to be installed
       Depends: node-archy but it is not going to be installed
       Depends: node-block-stream but it is not going to be installed
       Depends: node-fstream (>= 0.1.22) but it is not going to be installed
       Depends: node-fstream-ignore but it is not going to be installed
       Depends: node-github-url-from-git but it is not going to be installed
       Depends: node-glob (>= 3.1.21) but it is not going to be installed
       Depends: node-graceful-fs (>= 2.0.0) but it is not going to be installed
       Depends: node-inherits but it is not going to be installed
       Depends: node-ini (>= 1.1.0) but it is not going to be installed
       Depends: node-lockfile but it is not going to be installed
       Depends: node-lru-cache (>= 2.3.0) but it is not going to be installed
       Depends: node-minimatch (>= 0.2.11) but it is not going to be installed
       Depends: node-mkdirp (>= 0.3.3) but it is not going to be installed
       Depends: node-gyp (>= 0.10.9) but it is not going to be installed
       Depends: node-nopt (>= 3.0.1) but it is not going to be installed
       Depends: node-npmlog but it is not going to be installed
       Depends: node-once but it is not going to be installed
       Depends: node-osenv but it is not going to be installed
       Depends: node-read but it is not going to be installed
       Depends: node-read-package-json (>= 1.1.0) but it is not going to be installed
       Depends: node-request (>= 2.25.0) but it is not going to be installed
       Depends: node-retry but it is not going to be installed
       Depends: node-rimraf (>= 2.2.2) but it is not going to be installed
       Depends: node-semver (>= 2.1.0) but it is not going to be installed
       Depends: node-sha but it is not going to be installed
       Depends: node-slide but it is not going to be installed
       Depends: node-tar (>= 0.1.18) but it is not going to be installed
       Depends: node-underscore but it is not going to be installed
       Depends: node-which but it is not going to be installed
E: Unable to correct problems, you have held broken packages.

The complete log is attached 
[nodejs_error.txt](https://github.com/prasadtalasila/BITS-Darshini/files/1335045/nodejs_error.txt)

**Comment by prasadtalasila** [comment]
As per this [tutorial](https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-16-04), the commands are:

```shell
> curl -sL https://deb.nodesource.com/setup_6.x -o nodesource_setup.sh
> sudo bash nodesource_setup.sh
> sudo apt-get install nodejs -y
> sudo apt-get install build-essential
```

The curl command was represented incorrectly in your comment. I tried these instructions on the server and the software has been installed successfully without any error.

**Comment by prasadtalasila** [comment]
solved in 7a7c857.
